torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3457,11 +3461,12 @@ def large_1d_unique():
 
 def sample_inputs_threshold(op_info, device, dtype, requires_grad, **kwargs):
     make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+    make_scalar_arg = partial(make_tensor, dtype=dtype, device='cpu', requires_grad=requires_grad)
```

Comment:
Nit: this could pass the correct shape and take the `.item()` here so you wouldn't need to repeat yourself later